docs/website/docs/guides/developer-tips.md

```diff
@@ -435,3 +435,12 @@ $ iree-compile simple_exp_abi.mlir \
   --iree-hal-target-backends=llvm-cpu \
   -o simple_exp_cpu.vmfb
 ```
+
+or explicitly resume from an intermediate phase with `--compile-from=<phase name>`:
+
+```console
+$ iree-compile simple_exp_abi.mlir \
+  --iree-hal-target-backends=llvm-cpu \
+  --compile-from=flow \
```

Comment:
Shouldn't this use `--compile-from=abi` instead , matching the `--compile-to` above?

```suggestion
$ iree-compile simple_exp_abi.mlir \
  --iree-hal-target-backends=llvm-cpu \
  --compile-from=abi \
```

Reply:
It's because compile-from is inclusive, so the first phase it runs is the one specified. My interpretation of the phases was as a step of compilation, not an intermediate output point, thus compile-from should resume compilation starting at the specified phase. I can change it if you think it would be better to match the phase of `--compile-to`.

Reply:
Hmm... I think it's useful to be able to `--compile-to`, edit, `--compile-from` without needing to know which phase follows the phase you paused at (especially as that might change if/when a new phase is added)

Reply:
I think that as-written you can do a run with --compile-to=foo and then pick it up with a --compile-from=foo, right? (i.e. compile-to breaks and outputs before, compile-from starts there). I'd like to keep that correspondance since it matches the user need.

Reply:
No, compile-to will run the phase specified as well, so as written, to resume from where you left off we will have to specify the next pipeline. I wouldn't think we should change the compile-to behavior, so if that's the need then I'll make compile-from match compile-to. It does require adding a `start` compilation phase, although I don't think `IREEVMPipelinePhase` really refers to a phase anymore, rather it's more like an intermediate output format.

Reply:
Ok, let's make that change as it is much more ergonomic. There is already an `end` so why not have a `start`.

Reply:
Done! I made `--compile-to=foo` and `--compile-from=foo` in the same `iree-compile` command an error. I can instead let it go through as a no-op if that's better.

Reply:
Landed as is; if this should be allowed I can do it as a follow up.